v0/destinations/rockerbox/transform.js

```diff
@@ -29,7 +30,9 @@ const responseBuilderSimple = (message, category, destination) => {
   } else {
     payload.action = eventsHashMap[message.event.toLowerCase()];
   }
-
+  // mapping external id
+  const externalId = getDestinationExternalID(message, "rockerboxExternalId");
+  payload.external_id = externalId;
```

Comment:
Based on the Rockerbox team's response, it seems like we don't have to send the external ID.

Reply:
We have removed externalId for now, so that we can release with the other 2 supported fields.
CC @utsabc